package/yast2-network.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Mon Aug 21 07:02:32 UTC 2017 - lslezak@suse.cz
+
+- AutoYaST: Do not display a warning about disabled second stage
+  when the hostname setting is read out of profile (bsc#1054400)
```

Comment:
... and second stage is enabled

Reply:
No, the second stage is actually disabled, but the setting is read out of profile so it is not an user fault.

Reply:
ah ok.